libs/service-portal/applications/src/components/ApplicationCard/ApplicationCard.tsx

```diff
@@ -4,70 +4,102 @@ import {
   Stack,
   Text,
   Tag,
-  Button,
   Columns,
   Column,
 } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+
 import * as styles from './ApplicationCard.treat'
 import ProgressBar from '../ProgressBar/ProgressBar'
+import { m } from '../../lib/messages'
 
 interface Props {
   name: string
   date: string
-  isComplete: boolean
+  isApplicant?: boolean
+  isAssignee?: boolean
+  isComplete?: boolean
   url: string
   progress: number
 }
 
 const ApplicationCard: FC<Props> = ({
   name,
   date,
+  isApplicant,
+  isAssignee,
   isComplete,
   url,
   progress,
 }) => {
+  const { formatMessage } = useLocale()
+
   return (
     <Box
       className={styles.wrapper}
       paddingY={3}
       paddingX={4}
       border="standard"
       borderRadius="large"
+      style={{ opacity: isComplete ? 0.5 : 1 }}
     >
-      <Stack space={1}>
-        <Box
-          display="flex"
-          justifyContent="spaceBetween"
-          alignItems="center"
-          marginBottom={1}
-        >
-          <Text variant="h3" as="h3">
-            {name}
-          </Text>
-          <Tag variant={isComplete ? 'mint' : 'purple'} outlined>
-            {isComplete ? 'Lokið' : 'Í ferli'}
-          </Tag>
-        </Box>
-        <Text>
-          {`Þú hefur ${
-            !isComplete ? 'ekki ' : ''
-          } lokið umsóknarferli fyrir ${name}`}
-        </Text>
-        <Columns space={8} alignY="center" collapseBelow="md">
-          <Column width="8/12">
-            <ProgressBar progress={progress} />
-          </Column>
-          <Column width="4/12">
-            <Box display="flex" justifyContent="flexEnd">
-              <a href={url} target="_blank" rel="noopener noreferrer">
-                <Button variant="text">
-                  {isComplete ? 'Skoða umsókn' : 'Halda áfram'}
-                </Button>
-              </a>
-            </Box>
-          </Column>
-        </Columns>
-      </Stack>
+      <a href={url} target="_blank" rel="noopener noreferrer">
```

Comment:
Could use [Link](https://github.com/island-is/island.is/blob/main/libs/island-ui/core/src/lib/Link/Link.tsx#L90)